ChandraByte/hamro-nepse
# claculator added.

## Share price calculator added.
#24 

## Screenshots 

![image](https://github.com/ChandraByte/hamro-nepse/assets/62834029/224e44ba-d657-4cab-af60-7ebe6529978a)

## Testing instruction

npm run dev 
you can see it on the main page itself. 

##
further work to be done.
